html/RegistrationPage.html

```diff
@@ -17,19 +17,31 @@
             <div class="col sing_in_box">
                 <h1 class="mb-1 display-2">Registration !</h1>
                 <p class="mb-5"><small>Enter your account name and password :)</small></p>
-                <div class="input-group mb-3 ">
-                    <input type="text" class="form-control" placeholder="Account name" aria-label="Account name">
-                    <span class="input-group-text"> </span>
-                    <input type="password" class="form-control" placeholder="Password" aria-label="Password">
-                </div>
-                <div class="d-grid gap-2 col-6 mx-auto mt-4 public_shadow">
-                    <button class="btn btn-success" type="button">Registration</button>
+                <form id="singup" onsubmit="registration()">
```

Comment:
put _ between words for id
do the same for other places where u used single word